cli/src/pcluster/templates/queue_group_stack.py

```diff
@@ -133,12 +133,12 @@ def _add_launch_templates(self):
                     self._compute_instance_profiles,
                 )
 
-    def _get_custom_compute_resource_tags(self, queue_config):
+    def _get_custom_compute_resource_tags(self, queue_config, compute_resource_config):
         """Override Queue Tags value on Cluster level tags if there are duplicated keys."""
-        tags = get_custom_tags(self._config, raw_dict=True)
-        queue_tags = get_custom_tags(queue_config, raw_dict=True)
-        tags.update(queue_tags)
-        return dict_to_cfn_tags(tags)
+        tags = {**get_custom_tags(self._config, raw_dict=True)}
+        queue_tags = {**get_custom_tags(queue_config, raw_dict=True)}
+        compute_resource_tags = {**get_custom_tags(compute_resource_config, raw_dict=True)}
+        return dict_to_cfn_tags({**tags, **queue_tags, **compute_resource_tags})
```

Comment:
Is the function description ("_Override Queue Tags value on Cluster level tags if there are duplicated keys_") still accurate? Now it looks like compute resource tags will also replace the existing keys in cluster tags and queue tags?
BTW, what's the difference between `xx_tags = {**get_custom_tags(...)}` and `xx_tags = get_custom_tags(...)` here given they are the only dictionary whose contents get unpacked there?

Reply:
since get_custom_tags only return one dict, we don't need to unpacked it here, we only need the unpack for dict_to_cfn_tags({**tags, **queue_tags, **compute_resource_tags})
Function description is updated, thanks